testhelpers/registry.go

```diff
@@ -17,9 +16,10 @@ import (
 	dockertypes "github.com/docker/docker/api/types"
 	dockercontainer "github.com/docker/docker/api/types/container"
 	"github.com/docker/go-connections/nat"
+	"golang.org/x/crypto/bcrypt"
 )
 
-var registryContainerName = "registry:2"
+var registryContainerName = "cnbs/registry:2"
```

Comment:
Where's the source for this image?

Reply:
There is no dockerfile for it, per se. It's an image whose manifest list was adjusted to support both linux and windows daemons. It's essentially a manifest list that has the following:
- `stefanscherer/registry-windows`
- `registry:2`

Reply:
Getting some initial concerns about long-term maintenance. Do you mind writing up how this image was constructed, either in a GoDoc or we can add it to our wiki? I know it's probably just one or two commands if I remember correctly but wouldn't want to lose that context.

Reply:
Definitely valid concerns. We will definitely document this. Would you be willing to merge the PR as is and we can follow up with any documentation and/or refactors?

As an alternative to documentation, we can do what `imgutil` does and just use a hash map in the test code to select the proper image from the bullets above (see https://github.com/buildpacks/imgutil/blob/master/testhelpers/docker_registry.go#L19), and not have to maintain a manifest list ourselves. If you'd like to go that route, we can make that adjustment in a follow-up PR.

Reply:
The alternative has a lot less overhead, although not as clean. Would it be hard to change the implementation to that and omit the use of `cnbs/registry:2`?

Reply:
Done. We were going to do this in a follow-up PR, but @micahyoung snuck it in :)